### PR TITLE
Update animator CSS section

### PR DESCRIPTION
**Pull request by stephenmcgruer**

### CODE REVIEW

**Comment by stephenmcgruer** [comment]
@flackr This is just an FYI atm, I'm working on improving the CSS notation section of the spec.

**Comment by stephenmcgruer** [comment]
Ping for review?

File: Overview.bs [hunk 1]
```diff
@@ -487,30 +491,26 @@ Media: interactive
...
 
+      * If there is no <a>animator definition</a> registered with the given <<ident>>, the inclusion
+        of that <<ident>> as a value has no effect.
 
-Note: All the elements in the root element's DOM sub-tree that get associated with the same animator
- name will get assigned to the the animator instance linked with this root element.
+      * Otherwise, an <a>element proxy</a> for this element will be passed into the
+        <a>animate function</a> of the closest ancestor <a>animator instance</a> with the given
+        <<ident>> as one of the <a>children element proxies</a>. If no such <a>animator instance</a>
+        exists, one is created with the document root element as it's <a>root element proxy</a>.
```
Comment (flackr) [inline Overview.bs:509]: Except that I don't think we'll actually pass in a proxy for the document root. This should maybe just refer to the animator instance with no root element?

**Review by flackr — CHANGES_REQUESTED** [review]

File: Overview.bs (hunk 1 above)
Comment (flackr) [reply Overview.bs:509]: I've changed my mind on this one. Passing the root element is simple and easy to reason about.

**Review by flackr — APPROVED** [review]

**Pull request merged**